gemfiles/contrib.gemfile

```diff
@@ -3,13 +3,13 @@
 source "https://rubygems.org"
 
 gem "elasticsearch-transport"
+gem "rack"
```

Comment:
I'm surprised `contrib_old.gemfile` does not show up in the list of updated files. I suspect running appraisal with some older Ruby (say 1.9) should generate it.